_pages/infrastructure/good-production-practices.md

```diff
@@ -64,6 +64,8 @@ See [the page on monitoring](../monitoring/).
 - **Third-party services are approved to hold the data being sent to them**
 - Automated pen-testing in a staging environment as part of continuous deployment
 - Automated vuln-scanning in production environment that is fed with newly-discovered vulns
+- Enable [SSL](https://github.com/18f/https) for everything
+- Redirect http to https
```

Comment:
Maybe we should say this explicitly? "If you are using Federalist/cloud.gov, you get these for free."